client/src/pages/private/SiteTable.js

```diff
@@ -24,15 +25,37 @@ import { FeatureFlaggedComponent, flagKeys } from '../../services';
 import { fetchRegionSiteRows, fetchSiteRows } from '../../services/site';
 import { useTableStyles } from '../../components/tables/DataTable';
 
+const renderAllocations = (row) => {
+  const formattedDate = (date) => dayjs(row[date]).format('MMM D YYYY');
+  return (
+    <div>
+      {row['allocation'] || row['startDate'] ? (
+        <div>
+          <div style={{ fontWeight: '700' }}>{row['allocation']}</div>
+          <FeatureFlaggedComponent featureKey={flagKeys.FEATURE_PHASE_ALLOCATION}>
+            <div style={{ color: '#272833', fontWeight: '400' }}>
```

Comment:
Can we get a `makeStyles` in this page instead of inline?

Reply:
I have this function outside the react class in order for the `columns` to reference it, and I cannot use `makeStyles` outside of the component, do you want me to move the styling to a regular object?

Reply:
created a regular style object vs inline. Let me know if this works for this use case!